lib/rubocop/cop/style/numbered_parameters.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Use numbered parameters only for single-line block.
```

Comment:
I guess this needs to be updated to reflect the more general nature of the cop today.

Reply:
Updated, ok for you? (I updated default.yml as well)